Microsoft.Azure.Functions.PowerShellWorker.Common.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+  <PropertyGroup>
+    <BuildNumber Condition="$(APPVEYOR) != ''">$(APPVEYOR_BUILD_NUMBER)</BuildNumber>
+    <BuildNumber Condition="$(APPVEYOR) == ''">9999</BuildNumber>
```

Comment:
Can we just use the date if it's not built in appveyor?

Reply:
now using the date - could only use month and day because BuildNumber can only be 4 digits max.